README.rst

```diff
@@ -34,21 +37,18 @@ You will need third-party software such as fastqc. Please see below for details.
 Usage
 ~~~~~
 
-This command will scan all files ending in .fastq.gz found in the local
-directory, create a directory called fastqc/ where a snakemake pipeline is
-launched automatically. Depending on the number of files and their sizes, the
-process may be long::
+If you have a set of FastQ files in a data/ directory, type::
 
-    sequana_fastqc --run
+    sequana_fastqc --input-directory data
 
 To know more about the options (e.g., add a different pattern to restrict the
 execution to a subset of the input files, change the output/working directory,
 etc)::
 
     sequana_fastqc --help
-    sequana_fastqc --input-directory DATAPATH
 
-This creates a directory **fastqc**. You just need to execute the pipeline::
+The call to sequana_fastac creates a directory **fastqc**. Then, you go to 
```

Comment:
The call of* sequana_fastqc* creates ... /  calling sequana_fastqc creates a directory **fastqc**